packages/app-store/routing-forms/trpc/response.handler.ts

```diff
@@ -91,7 +91,10 @@ export const responseHandler = async ({ ctx, input }: ResponseHandlerOptions) =>
     }
 
     const dbFormResponse = await prisma.app_RoutingForms_FormResponse.create({
-      data: input,
+      data: {
+        formId,
+        response: response,
```

Comment:
Be specific with what you send to data. Now, there is a prop in input that doesn't actually part of FormResponse schema